data/schema/filters/abilities.cfg

```diff
@@ -2,6 +2,17 @@
 [tag]
 	name="$filter_abilities"
 	max=0
+	super="$filter_abilities_common"
```

Comment:
Why was this split in two? I'm not complaining about it or anything, I just want to know the reason.

Reply:
in [specials] attributes like affect_self/allies tec don't exist. It should be ignored in src ans schema because no valid

Reply:
Okay, so it sounds like `$filter_abilities` should be `$filter_specials`, and the other one should be `$filter_abilities`… though I'm not sure if there's still a reason to have a third one that the two share in common.